### PR TITLE
Preservation Evoker - Mastery Healing Module

### PR DESCRIPTION
**Pull request by tyndivelspaz**
Added a module to calculate the healing "% Effected by Mastery"

image

This module will take into account all healing and absorbs from the Evoker.

### CODE REVIEW

File: src/analysis/retail/evoker/preservation/modules/core/MasteryEffectiveness.tsx [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  statistic() {
+    return (
+      <Statistic
+        size="flexible"
+        position={STATISTIC_ORDER.CORE(10)}
+        category={STATISTIC_CATEGORY.GENERAL}
+      >
+        <div className="pad">
+          <label>
+            % of Healing Effected by <SpellIcon id={SPELLS.MASTERY_LIFEBINDER.id} />
```
Comment (trevorm4) [inline src/analysis/retail/evoker/preservation/modules/core/MasteryEffectiveness.tsx]: Affected*

File: src/analysis/retail/evoker/preservation/CHANGELOG.tsx [hunk 2]
```diff
@@ -2,5 +2,6 @@ import { change, date } from 'common/changelog';
 import { Tyndi } from 'CONTRIBUTORS';
 
 export default [
+  change(date(2022, 10, 19), 'Added module tracking healing effected by mastery.', Tyndi),
```
Comment (ToppleTheNun) [inline src/analysis/retail/evoker/preservation/CHANGELOG.tsx]: This one is also `affected`.

**Review by ToppleTheNun — APPROVED** [review]
One tiny comment, otherwise looks good to me.

File: src/analysis/retail/evoker/preservation/CHANGELOG.tsx (hunk 2 above)
Comment (tyndivelspaz) [reply src/analysis/retail/evoker/preservation/CHANGELOG.tsx]: darn it! Haha!

Fixed!

**Review by trevorm4 — APPROVED** [review]

**Review by kfinch — APPROVED** [review]

**Pull request merged**